Student.rb

```diff
@@ -3,7 +3,7 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+  def initialize(age, classroom = 5, name = 'Unknown', parent_permission: true)
     super(name, age, parent_permission)
     @classroom = classroom
   end
```

Comment:
Great job @29td 🤝

- [ ] Kindly add a `belongs-to` method `(a student belongs to a classroom)`. Check this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more guidance.